bus-mapping/src/evm/opcodes.rs

```diff
@@ -428,12 +428,12 @@ pub fn gen_begin_tx_ops(state: &mut CircuitInputStateRef) -> Result<ExecStep, Er
     ) {
         // 1. Creation transaction.
         (true, _, _) => {
-            state.account_write(
+            state.account_read(
                 &mut exec_step,
                 call.address,
                 AccountField::CodeHash,
-                call.code_hash.to_word(),
-                call.code_hash.to_word(),
+                Word::zero(),
+                Word::zero(),
```

Comment:
the address may not be non-exist.   For example, i can pre-calculate the contract address i will deploy into, then i first transfer some balance into that account to make it becomes no longer non-exist..